app/assets/stylesheets/dashboard.scss

```diff
@@ -355,19 +355,8 @@
     }
   }
 
-  .has-tip {
-    border-bottom: 0;
-    cursor: auto;
-    font-weight: normal;
-  }
-
   .submenu-active {
     border-bottom: 2px solid;
-
-    .has-tip {
-      @include brand-color;
-      font-weight: bold;
-    }
```

Comment:
This code might still be used in the `dashboard/menu` partial, since Foudation adds the `has-tip` class to elements with a `data-tooltip` attribute.